store/def_query_tree_index.go

```diff
@@ -43,6 +43,11 @@ func (x *defQueryTreeIndex) getByQuery(q string) (map[unit.ID2]byteOffsets, bool
 		panic("mafsaTable not built/read")
 	}
```

Comment:
I know there's a `if x.mt == nil {` guard just above, and it panics, so clearly this func is not meant to be called when `x.mt.t` is nil. That's why it's a short term fix.